### PR TITLE
feat(chain): support wasm api read solana tx

### PR DESCRIPTION
**Pull request by huangzhiran**

### CODE REVIEW

File: cmd/srv-applet-mgr/global/config.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func init() {
 
 	redisKvDB := kvdb.NewRedisDB(config.Redis)
 
-	wasmApiServer, err := wasmapi.NewServer(std, config.Redis, config.Postgres, redisKvDB, config.EthClient)
+	wasmApiServer, err := wasmapi.NewServer(std, config.Redis, config.Postgres, redisKvDB, config.EthClient, config.ChainConfig)
```
Comment (hunshenshi) [inline cmd/srv-applet-mgr/global/config.go:144]: still need `ethClient`?

File: pkg/modules/vm/wasmapi/handler/read_tx.go [hunk 2]
```diff
@@ -7,21 +7,29 @@ import (
...
 }
 
-type readTxResp struct {
+type readEthTxResp struct {
 	Transaction *ethtypes.Transaction `json:"transaction,omitempty"`
 }
 
+type readSolanaTxResp struct {
+	Transaction *rpc.GetTransactionResult `json:"result,omitempty"`
+}
```
Comment (hunshenshi) [inline pkg/modules/vm/wasmapi/handler/read_tx.go:31]: `result` should be `transacton`, same with `readEthTxResp`

File: cmd/srv-applet-mgr/global/config.go (hunk 1 above)
Comment (huangzhiran) [reply cmd/srv-applet-mgr/global/config.go:144]: will delete in next pr. "support solana sendTX"

File: pkg/modules/vm/wasmapi/handler/read_tx.go (hunk 2 above)
Comment (huangzhiran) [reply pkg/modules/vm/wasmapi/handler/read_tx.go:31]: https://docs.solana.com/api/http#sendtransaction

**Review by hunshenshi — APPROVED** [review]

**Pull request merged**